src/common/mfu_flist_copy.c

```diff
@@ -3220,8 +3220,8 @@ mfu_file_t* mfu_file_new(void)
     mfile->fd         = -1;
 #ifdef DAOS_SUPPORT
     mfile->obj        = NULL;
+    mfile->dfs_sys    = NULL;
     mfile->dfs        = NULL;
```

Comment:
@daltonbohning is mfile->dfs still used? It looks like it has been changed to mfile->dfs_sys